rust/core-lib/src/client.rs

```diff
@@ -188,6 +188,16 @@ impl Client {
         ))
     }
 
+    pub fn handle_definition(&self, view_id: ViewId, request_id: usize, result: Vec<Location>) {
+        self.0.send_rpc_notification("show_hover", &json!(
```

Comment:
This is meant to be "show_definition" right?

Reply:
Thanks for pointing out. 👍  Fixed.